tests/Import-DbaCsv.Tests.ps1

```diff
@@ -30,9 +31,10 @@ Describe "$CommandName Integration Tests" -Tag "IntegrationTests" {
             $results | Should -Be $null
         }
 
-        It "creates the right columnmap (#7630)" {
+        It "creates the right columnmap (#7630), handles pipe delimiters (#7806)" {
             $null = Import-DbaCsv -SqlInstance $script:instance1 -Path $col1 -Database tempdb -AutoCreateTable -Table cols
             $null = Import-DbaCsv -SqlInstance $script:instance1 -Path $col2 -Database tempdb -Table cols
+            $null = Import-DbaCsv -SqlInstance $script:instance1 -Path $pipe3 -Database tempdb -Table cols -Delimiter "|"
```

Comment:
This is a false positive test considering `$pipe3` has no content from the Appveyor run.

Reply:
also true, i'll rerun the test but it seems it is indeed a false positive